ui/lib/theme.ts

```diff
@@ -170,5 +170,11 @@ export const muiTheme = createTheme({
         overflowX: "hidden",
       },
     },
+    MuiDrawer: {
+      paper: {
+        width: "60%",
+        minWidth: 600,
```

Comment:
How does this look on a laptop screen vs a desktop screen? Mobile?

Reply:
Here's a screen from a huge width - on small screens in OSS there'a min-width for the entire app that intentionally prevents any attempts at responsiveness - this doesn't exist in enterprise which is a much larger thing outside of this, but for now I've been told to not work on any views for mobile
![image](https://user-images.githubusercontent.com/65822698/222172171-f05f3020-21ad-4ac0-be08-a970d0fc4181.png)